### DISCUSSION TITLE
Support for zones?

### DISCUSSION THREAD

**Discussion by tiprya** [revk/xESP32-Faikout#391] [Q&A]
Hi - thanks very much for this project, amazed at how well it works :)

I have a ducted system for two zones (upstairs and down) that is currently controlled by a zone controller on the wall.

Will faikin work with this setup, i.e. enable controlling both zones separately?
If so, will it expose two separate sets of controls in HA, for example?

Apologies if this is in the documentation somewhere and I missed it!

**Comment by campinge** [comment]
I‘m seconding this here. We are in the process of buying a ducted system. It would be great to understand if there already is a way to control AZEZ multizone kits?

**Comment by judo16** [comment]
Just wondering if anyone ended up getting an answer to this? I couldn't find it in the documenation either, and with my Daikin wifi module playing up, I'm hoping that this unit will be able to repalce it. My dampers are running through Daikin currently, not a 3rd party controller.

**Comment by revk** [comment]
We just need to know the commands, e.g. X50A or S21, do to them, and I'll be happy to implement.

**Reply by judo16** [reply]
Hi revk, thanks for your prompt response. I'd be happy to help with this if I can. I'll be using the X50A port. I'm purchasing the cable and module now. Please let me know if there's a way to capture the data ie a developer mode or something on the module?

**Reply by revk** [reply]
The issue is you ideally need an official WiFi module and tap what it sends and received on the serial line.

**Reply by judo16** [reply]
I'm currently using the Daikin BRP15A61, but that appears to use the P1 and P2 terminals. I'm not sure is there's serial terminals available on the board?
![20250623_170752 - Copy](https://github.com/user-attachments/assets/7c32670b-b1e3-4c2a-b7ee-3fb7ffcc4259)
![20250623_170952 - Copy](https://github.com/user-attachments/assets/d3646768-244c-4f93-9dac-e1093f8a896e)

**Reply by revk** [reply]
We can't do P1/P2, sorry.

**Reply by judo16** [reply]
Is there another way to test and understand the zone control? I'm assuming that the Faikin device would receive updated zone information when the zones are controlled?

**Reply by revk** [reply]
The Faikin polls specific status codes. Others have looked and not seen anything in these for zones.

Without knowing the codes to poll, it won't see anything.

**Comment by hole222** [comment]
Follow the P1 and P2 cable back to the main board. You may find an X50A or S21 connector there. MY zoneless ducted system (FDYA100AV1) has the X50A connector.

**Reply by judo16** [reply]
Yeah, the main board definitely has X50A and X35A connectors. That's not the issue. They'll drive the bulk of the system, however there's no details on how to drive the dampers from it. I'm using the Daikin BRC24Z8 zone controller which is drivable through the Daikin app using the Daikin BRP15A61, however it's slow and unreliable. I was hoping to upgrade to the Faikin module.

**Reply by revk** [reply]
if it connects to the X50A it may be possible to tee off its GND/PWR and a Tx line in to Faikin Rx, and then set Faikin X50A only and debug, and it will dump what it sees. Then we do same tapped on to the Rx line, and see what we get. That may be a way to find out.

**Reply by judo16** [reply]
I'm all for testing and seeing what the other headers on the BRP15A61 do. In the photos above, do any of them appear to potentially X50A connections? I don't know if there is a way to capture what is happening on the P1P2 terminals and see if it aligns with the known commands on the X50A port?

There are a few other projects where others have connected to the P1P2 and successfully read data. I don't know if they are of any help to see if the commans align with what Faikin is using?

https://github.com/opensensor/Daikin-P1P2---UDP-Gateway
https://forum.arduino.cc/t/reading-data-on-daikin-rotex-p1-p2-control-bus-possible/580520
https://github.com/Arnold-n/P1P2MQTT

**Reply by revk** [reply]
P1/P2 is very different

**Reply by judo16** [reply]
Does anyone know what the equivalent official Daikin module is? BRP069C41? Is it still available?
